evals/README.md

```diff
@@ -8,15 +8,6 @@ Source code:
 ## Step 1: Pull Docker image
 
 Docker images are published to: https://ghcr.io/tenstorrent/tt-inference-server/tt-metal-llama3-70b-src-base-vllm
-```bash
-# commit SHAs
-export TT_METAL_COMMIT_SHA_OR_TAG=685ef1303b5abdfda63183fdd4fd6ed51b496833
```

Comment:
now that the tt-metal and vllm commit sha are removed here, do we need to specify where someone can find what commit sha to use (ie. point back to vllm-tt-metal-llama3-70b/src or the vllm fork)?

Reply:
It's in the main readme for vllm-tt-metal-llama3-70b: https://github.com/tenstorrent/tt-inference-server/tree/main/vllm-tt-metal-llama3-70b#option-b-build-docker-image

This was just duplicating it and became outdated.

Reply:
I improved the docker build documentation 👍